tests/CMakeLists.txt

```diff
@@ -66,6 +65,8 @@ function(gen_test name files compile_definitions compile_flags link_flags)
 endfunction()
 
 function(gen_test_matrix name files compile_definitions)
+    gen_test("${name}_c99" "${files}" "${compile_definitions}" "-std=c99" "-std=c99")
+    gen_test("${name}_c11" "${files}" "${compile_definitions}" "-std=c11" "-std=c11")
```

Comment:
1. Maybe I am missing something but as far as I know `-std` is not a valid linker option.
2. We need a Cartesian product of all possible options; it costs us nothing to add a few more lines to cover them all:
  - 64-bit C11
  - 32-bit C11
  - 64-bit C99
  - 32-bit C99

Reply:
Yeah, this is the right approach. I stopped a few hours ago because I couldn't wrap my head around how I should go about this, turns out we're thinking the same thing (reflected in my recent push).

Gosh I wish we can make the build time shorter...

Reply:
> Gosh I wish we can make the build time shorter...

You should build it locally. Much faster this way.